### DISCUSSION TITLE
10x Multiome without reference

### DISCUSSION THREAD

**Discussion by MallipattuLab** [stuart-lab/signac#727] [Q&A] [answered]
Running 10x multiome on tissue where no reference yet exists. What would be appropriate steps to annotating cell types for this new multiomic data, doing RNA and ATAC alone and then merging? Thanks.

**Comment by timoast** [comment, marked as answer]
To annotate the cells in a multiome dataset, without reference mapping, you can use all the same methods you'd use to annotate a new scRNA-seq dataset (find clusters, find differential expressed genes for each cluster, read the literature to find what cell types express those genes, perform manual cell type annotation)
